Source/WebCore/svg/svgattrs.in

```diff
@@ -71,7 +71,6 @@ g2
 glyph-name
 glyph-orientation-horizontal
 glyph-orientation-vertical
-glyphRef
```

Comment:
When I removed `animateColor, it started failing HTML5 Parser tests and have to do:

`animateColor interfaceName=SVGUnknownElement, JSInterfaceName=SVGElement, noConstructor`

Till this not get fixed in tests upstream (Anne did I think), we can do this workaround.

Refer - https://commits.webkit.org/276683@main

Reply:
I would personally patch it locally first (rather than fixing tests) and then see which tests are failing genuinely (besides HTML5 Parser and whether they get fixed with this).